oviewer/oviewer.go

```diff
@@ -426,8 +426,8 @@ var (
 	ErrNotAlignMode = errors.New("not an align mode")
 	// ErrNoColumnSelected indicates that no column is selected.
 	ErrNoColumnSelected = errors.New("no column selected")
-	// ErrInvalidCSI indicates that the CSI is invalid.
-	ErrInvalidCSI = errors.New("invalid CSI")
```

Comment:
Are you sure no one use you as a lib?

You should set it to ErrInvalidSGR (or keep it as is)

And add a `// Deprecated:` flag in the godoc

Reply:
I added this 2 days ago, so I'm pretty sure no one is using this.